js/script.js

```diff
@@ -13,16 +13,31 @@ const envImageUrl = `${baseURL}assets/images/`;
 
 const audioFiles = {};
 
-if (document.querySelector(".how-to-play-button")) {
-  const howToPlayButton = document.querySelector(".how-to-play-button");
-  howToPlayButton?.addEventListener("click", function () {
-    popupDialog.showModal();
-  });
-}
-const popupDialog = document.querySelector(".popup-dialog");
-const closePopupButton = document.querySelector(".close-popup-button");
+// if (document.querySelector(".how-to-play-button")) {
+//   const howToPlayButton = document.querySelector(".how-to-play-button");
+//   howToPlayButton?.addEventListener("click", function () {
+//     popupDialog.showModal();
+//   });
+// }
+// const popupDialog = document.querySelector(".popup-dialog");
+// const closePopupButton = document.querySelector(".close-popup-button");
+
+// closePopupButton.addEventListener("click", function () {
+//   popupDialog.close();
+// });
+
+
+/**
+ * How to play instructions
+ */
+const howToPlayButton = document.querySelector('.how-to-play-button');
+const closePopupButton = document.querySelector('.close-popup-button');
+
+howToPlayButton.addEventListener('click', function() {
+  popupDialog.showModal();
+});
```

Comment:
Issue (Blocking): In this one, you'll need to keep the `if (document.querySelector(".how-to-play-button"))` because there are pages that won't have it. If it is not there and JS looks for it, then it will break the JavaScript. Same for the close popup button, which currently does not have the check, which broke the Wordle game (because it didn't have a modal dialog).